stable/database/files/nuobackup

```diff
@@ -0,0 +1,161 @@
+#!/bin/sh
+#
+# simple script to automate backup
```

Comment:
It's not very simple, isn't it?
Can we have longer description?

Reply:
I will augment the comments.